docs/plugins/plugin-script.asciidoc

```diff
@@ -279,3 +285,53 @@ The default location of the `plugins` directory depends on which package you ins
 * {ref}/zip-windows.html#windows-layout[Directory layout of Windows `.zip` archives]
 * {ref}/deb.html#deb-layout[Directory layout of Debian package]
 * {ref}/rpm.html#rpm-layout[Directory layout of RPM]
+
+
+[discrete]
```

Comment:
Right now, this content is appended to the [Plugins directory page](https://elasticsearch_80760.docs-preview.app.elstc.co/guide/en/elasticsearch/plugins/master/_plugins_directory.html), which seems odd to me. I'd drop the `[discrete]` tag so it renders on its own page.
```suggestion
```